rpm/gmime.spec

```diff
@@ -3,16 +3,12 @@ Name:       gmime
 %define keepstatic 1
```

Comment:
Hmmp... I wonder if we use this anywhere, would drop if not.

Reply:
Defined whether the .a file gets installed, but let's actually get rid of both. We haven't really used those anyway.